components/webui/server/src/configConstants.ts

```diff
@@ -0,0 +1,9 @@
+// NOTE: These settings are duplicated from components/webui/client/src/config/index.ts, but will be
+// removed in the near future.
+const CLP_STORAGE_ENGINE_CLP_S = "clp-s";
```

Comment:
is it better to use an enum like
```ts
enum CLP_STORAGE_ENGINE {
   CLP: "clp",
   CLP_S: "clp-s",
}
```

then compare the values against

Reply:
probably better to move stuff from this file into the typings directory

Reply:
We have this enum in the client code. This constant is just temporary to keep the webui working in this PR. In #1050, we're going to pass the dataset from the client code, so the logic int he last two commits can be reverted.

Reply:
Or are you suggesting we duplicate the entire enum in the serve code for now?

Reply:
right i was going to suggest reusing / duplicating this enum: https://github.com/haiqi96/clp_fork/blob/7b42568fa9d335ecbe85f13edc5d3c111b4e8587/components/webui/client/src/config/index.ts

> so the logic in the last two commits can be reverted.

sure. in that case i think we can temporarily leave the constant as-is then